nix/workbench/backend/nomad/cloud.nix

```diff
@@ -23,6 +23,19 @@ let
     ]
   ;
 
+  validateNodeSpecs = { nodeSpecsValue }:
+    let
+      # SRE is using these 3 Nomad "datacenters" (how they are called in Nomad)
+      datacenters = [ "eu-central-1" "eu-west-1" "us-east-2" ];
```

Comment:
@fmaste, we need SRE to enable support of the original locations -- which were originally picked by benchmarking:

https://github.com/input-output-hk/cardano-node/blob/master/bench/cardano-topology/cardano-topology.hs#L367-L369

You might need to work with @mgmeier to negotiate this extension with SRE.